servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -812,31 +781,7 @@
             "description": "The report of the node health",
             "required": true,
             "schema": {
-              "type": "object",
-              "description": "The report of the node health",
-              "properties": {
-                "SourceId": {
-                  "type": "string"
-                },
-                "Property": {
-                  "type": "string"
-                },
-                "HealthState": {
-                  "$ref": "#/definitions/HealthState"
-                },
-                "Description": {
-                  "type": "string"
-                },
-                "TimeToLiveInMilliSeconds": {
-                  "type": "string"
-                },
-                "SequenceNumber": {
-                  "type": "string"
-                },
-                "RemoveWhenExpired": {
-                  "type": "boolean"
-                }
-              }
+              "$ref": "#/definitions/NodeHealthReport"
```

Comment:
They are similar today, but they may change in the future, so our c++ side have all of them separately. We have a requirement that to support backward compatibility. If in the future these models do not look the same, we want to make sure customers upgrade from older version do not break the build.